app/views/pages/publishers.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :page_title do %>
+  Publishers -
+<% end %>
+
+<main role="main" id="content">
+  <%= render partial: 'layouts/feedback_survey_banner' if flash[:beta_message] == 'hide' %>
+
+  <h1 class="heading-large">
+    Publish your data
+  </h1>
+  <div class="text">
+    <p>We’re changing the publishing tool and plan to launch a new improved service by June 2018. Until then, publishers can sign in using their current username (or email address).</p>
+
+    <p>New and changed datasets will appear as normal on the newly-designed data.gov.uk, now called the Find open data service.</p>
+
+    <h2 class="heading-medium">
+      Become a publisher
+    </h2>
+    <p>To get started, take a look at the <%= link_to 'information about publishing datasets', 'http://guidance.data.gov.uk/publishing_on_data_gov_uk_overview.html', target: '_blank' %>. You can also <%= link_to 'ask us for help', support_path %></p>
```

Comment:
We should link to the secure version of this page – https://guidance.data.gov.uk/publishing_on_data_gov_uk_overview.html